dali/pipeline/util/thread_pool.cc

```diff
@@ -25,7 +27,9 @@ ThreadPool::ThreadPool(int num_thread, int device_id, bool set_affinity)
     work_complete_(true),
     active_threads_(0) {
   DALI_ENFORCE(num_thread > 0, "Thread pool must have non-zero size");
+#if !defined(__AARCH64_QNX__) && !defined(__AARCH64_GNU__)
   nvml::Init();
```

Comment:
Maybe you could hide this #ifdef inside nvml?

Reply:
I am not sure if it is a great idea to move the flags inside the nvml.h
It will look like masking every function blocks with #ifdef  
Let me know your thoughts on this

Reply:
@JanuszL : Do you still want me to mask all the functions in nvml.h?